### PR TITLE
sync garbage worms

### PR DESCRIPTION
**Pull request by youbitchoc**

### CODE REVIEW

**Comment by 6fears7** [comment]
No complaints here, good stuff

File: Online/State/RealizedGarbageWormState.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            this.showAsAngry = garbageWorm.AI.showAsAngry;
+            this.lookPoint = garbageWorm.lookPoint;
+        }
+
+        public override void ReadTo(OnlineEntity onlineEntity)
+        {
+            base.ReadTo(onlineEntity);
+
+            var garbageWorm = (GarbageWorm)((OnlinePhysicalObject)onlineEntity).apo.realizedObject;
+            garbageWorm.AI.retractCounter = 0;
```
Comment (6fears7) [inline Online/State/RealizedGarbageWormState.cs:34]:
```
            garbageWorm.AI.retractCounter = 0;
            garbageWorm.AI.comeBackOutCounter = 0;
            garbageWorm.chargePos = null;
```

Just want to verify having these properties zeroed out is intended

Comment (youbitchoc) [reply Online/State/RealizedGarbageWormState.cs:34]: yup, prevents the worm from acting on its own

**Comment by youbitchoc** [comment]
yup, merging

**Pull request merged**